python/lsst/afw/math/pixelScaleBoundedField.cc

```diff
@@ -0,0 +1,66 @@
+/*
+ * This file is part of afw.
+ *
+ * Developed for the LSST Data Management System.
+ * This product includes software developed by the LSST Project
+ * (https://www.lsst.org).
+ * See the COPYRIGHT file at the top-level directory of this distribution
+ * for details of code ownership.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+
+#include "ndarray/pybind11.h"
+
+#include "lsst/pex/config/python.h"    // defines LSST_DECLARE_CONTROL_FIELD
+#include "lsst/afw/table/io/python.h"  // for addPersistableMethods
+
+#include "lsst/afw/geom/SkyWcs.h"
+#include "lsst/afw/math/BoundedField.h"
+#include "lsst/afw/math/PixelScaleBoundedField.h"
```

Comment:
Are there guidelines on what should/shouldn't be listed in the `#include` list? I was going with "explicit is better than implicit", but that's a python quote, so...

Reply:
I don't know if there are firm guidelines, but I generally try to keep the number of includes to a minimum. In this case it's just a distraction because the include guards should keep these extra includes from causing any trouble, but in general minimising the number of includes minimises the compile time.